### PR TITLE
New package: corkscrew-2.0

### PR DESCRIPTION
**Pull request by pulux**

### CODE REVIEW

File: srcpkgs/corkscrew/template [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Template file for 'corkscrew'
+pkgname=corkscrew
+version=2.0
+revision=1
+build_style=gnu-configure
+short_desc="Corkscrew is a tool for tunneling SSH through HTTP proxies"
+maintainer="pulux <pulux@pf4sh.de>"
+license="GPL-2"
+homepage="http://www.agroman.net/corkscrew/"
```
Comment (Gottox) [inline srcpkgs/corkscrew/template]: That's a 404 page

**Comment by pulux** [comment]
Hm, and why Travis and I hasn't a problem with that page?

**Comment by ayghor** [comment]
> Hm, and why Travis and I hasn't a problem with that page?

Idk, but I get 404 too. Also, http://www.agroman.net/ gives "Welcome to nginx!".


-- 
Igor

File: srcpkgs/corkscrew/template (hunk 1 above)
Comment (Vaelatern) [inline srcpkgs/corkscrew/template]: Not any more.

**Pull request merged**